plugins/test/framework.cc

```diff
@@ -62,10 +62,14 @@ uint64_t TestContext::getMonotonicTimeNanoseconds() {
   return absl::ToUnixNanos(options().clock_time);
 }
 proxy_wasm::WasmResult TestContext::log(uint32_t log_level,
-                                        std::string_view message) {
+  std::string_view message) {
   logging_bytes_ += message.size();
   if (wasmVm()->cmpLogLevel(proxy_wasm::LogLevel::trace)) {
-    std::cout << "TRACE from testcontext: [log] " << message << std::endl;
+    std::cout << "TRACE from integration: [vm->host] env.proxy_log(" 
+    << log_level << ", " 
+    << reinterpret_cast<uint64_t>(message.data()) << ", " 
```

Comment:
I'm not sure how useful it is to display the address and size of the message string, given that the value of the string is already output. Did you have a case in mind where this information would be valuable?

Reply:
Friendly ping on this PR. I had a use for it today (testing/inspecting a plugin I did not write and trying to understand its behaviors).